cpp/src/io/utilities/config_utils.hpp

```diff
@@ -30,6 +30,41 @@ inline std::string getenv_or(std::string const& env_var_name, std::string_view d
   return std::string{(env_val == nullptr) ? default_val : env_val};
 }
 
+namespace cufile_integration {
+
+namespace {
```

Comment:
Anonymous namespaces should not be used in a header. [C++ Core Guidelines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-unnamed)

Reply:
moved all implementations to a .cpp file